test/envs/regional-dr-argocd.yaml

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Environment for testing Regional-DR.
+---
+name: "rdr"
```

Comment:
This clashes with regionl-dr.yaml - pick a unique name.

Reply:
this is a temp file wrongly uploaded. Already deleted it.